client/static/style.css

```diff
@@ -0,0 +1,41 @@
+body{
+    font-family: 'Lato', sans-serif;
+}
+
+img{
+    width: 400px;
+    margin: 1% 37% 1% 37%;
+}
+li{
+    list-style: none;
+}
+
+#formContent{
+    display:flex;
+    justify-content: center;
+    padding: 4px;
+}
+
+form{
+    display:flex;
+}
+#results ul{
+    text-align: center;
+    border-radius: 1rem;
+    margin-top: 2rem;
+    background-color:rgb(245, 241, 241);
+    margin-left:10rem;
+    margin-right:10rem;
+    
+}
+#results li{
+    margin-top: 1rem;
```

Comment:
I think you could change this to `margin-top: 1rem 4rem 1rem 4rem` to shorten the code maybe have a go?